manifests/h/HydrusNetwork/HydrusNetwork/467/HydrusNetwork.HydrusNetwork.locale.en-US.yaml

```diff
@@ -0,0 +1,33 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=NVS1.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: HydrusNetwork.HydrusNetwork
+PackageVersion: "467"
+PackageLocale: en-US
+Publisher: Hydrus Network
+PublisherUrl: https://github.com/hydrusnetwork/hydrus
+PublisherSupportUrl: https://github.com/hydrusnetwork/hydrus/issues
+# PrivacyUrl: 
+Author: Hydrus Network
+PackageName: Hydrus Network
+PackageUrl: https://github.com/hydrusnetwork/hydrus
+License: Do What the Fuck You Want To Public License
```

Comment:
This license fails our policycheck. The licenseUrl is sufficient, just remove this line

```suggestion
```

Reply:
Removing the line would fail the validation, the License field is required :)
I will use the short term: WTFPL

Reply:
Thanks @OfficialEsco